numpy/core/src/multiarray/descriptor.c

```diff
@@ -2391,7 +2398,10 @@ arraydescr_reduce(PyArray_Descr *self, PyObject *NPY_UNUSED(args))
         }
         obj = PyUString_FromFormat("%c%d",self->kind, elsize);
     }
-    PyTuple_SET_ITEM(ret, 1, Py_BuildValue("(Nii)", obj, 0, 1));
+    PyTuple_SET_ITEM(ret, 1, Py_BuildValue("(Nii)", obj, 0, !builtin));
```

Comment:
Hmm, builtins do have endianess. Does builtin have cross platform implications? I suppose that depends on how things get unpickled. Or does the endianess have anything to do with the builtin designation? Perhaps I am confusing builtin with native.